File: 26.remove-duplicates-from-sorted-array.java
/*
 * @lc app=leetcode id=26 lang=java
 *
 * [26] Remove Duplicates from Sorted Array
 */

// @lc code=start
import java.util.Arrays;

class Solution {
    public int removeDuplicates(int[] nums) {
        int k =-101;
        int p =0;
        for (int i = 0; i < nums.length; i++) {
                if(k!=nums[i]){
                    k=nums[i];
                    p++;
                }else{
                    nums[i]=101;
                }

        }
        Arrays.sort(nums);
return p;
    }
}
// @lc code=end

File: 26.remove-duplicates-from-sorted-array.py
#
# @lc app=leetcode id=26 lang=python3
#
# [26] Remove Duplicates from Sorted Array
#

# @lc code=start
from typing import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        k=-101
        p=0
        i =0
        while i<len(nums):
            if k!=nums[i]:
                k=nums[i]
                p+=1
            else:
                nums.remove(k)
                nums.append(101)
            i+=1    
        return p
# @lc code=end

